keras_cv/tools/convert_presets.py

```diff
@@ -95,7 +105,7 @@ def convert_task_presets():
         )
         for preset in task_preset_keys:
             save_weights = preset in task_cls.presets_with_weights
-            kwargs = {}
+            kwargs = {"name": to_snake_case(task_cls.__name__)}
```

Comment:
Will this not hit the else branch below? pull `task = task_cls.from_preset(preset, **kwargs)` out of the if/else and leave on `kwargs.udpate` in the if.

Reply:
Ah my mistake, fixed!